### PR TITLE
Wording for FAQ and landing page

### PR DESCRIPTION
**Pull request by mcdurdin**
Now ready for review.

Previously: The FAQ page is not yet routed; can I leave that for you, @SeriousMatt? There are a bunch of incomplete references etc in that file which I am working on clarifying, but I wanted to get the basic structure there.

### CODE REVIEW

**Review by PaulNel — APPROVED** [review]
Changes look good to me. Thanks for putting these together!

**Comment by mcdurdin** [comment]
Note: this is not ready for merge yet; I still need to fixup links.

**Review by SeriousMatt — APPROVED** [review]
@mcdurdin The routing for the faq page (/faq) is now set up.

**Comment by mcdurdin** [comment]
@SeriousMatt this is now ready for review; added header, footer, ways-to-give, tidied up formatting of FAQ, fixed links and content, added very basic navigation. I think it's pretty much good enough to go now.

**Review by SeriousMatt — APPROVED** [review]
@mcdurdin works locally for me and passing specs. I'll merge this in and get it deployed.

File: src/app/app.component.ts [hunk 1]
```diff
@@ -9,219 +9,6 @@ import { Component, OnInit } from '@angular/core';
...
-        const apiURL = 'https://donate.keyman.com/api/charge';
-
-        self.http.post(apiURL, postData, {
-          headers: new HttpHeaders().set('Content-Type', 'application/json'),
-          observe: 'response', // TODO How do we catch 4xx errors?
-        }).subscribe(
-          success => {
-            // Handle result
-            console.log('POST call successful value returned in body', success);
-            self.successMessage = 'The transaction has been completed. Thank you for you donation!';
```
Comment (SeriousMatt) [inline src/app/app.component.ts:68]: @mcdurdin here is some the missing code from the merge. This occurred because we changed file names.

**Pull request merged**